### PR TITLE
Added docker-gc parameter

### PR DESCRIPTION
**Pull request by joel-hamill**

### CODE REVIEW

**Comment by joel-hamill** [comment]
blocked by https://github.com/dcos/dcos/pull/683

**Comment by joel-hamill** [comment]
@cmaloney should this be merged?

File: 1.8/administration/installing/custom/configuration-parameters.md [hunk 1]
```diff
@@ -176,6 +176,12 @@ This parameter specifies whether to check if Network Time Protocol (NTP) is enab
 ### <a name="docker-remove"></a>docker_remove_delay
 This parameter specifies the amount of time to wait before removing stale Docker images stored on the agent nodes and the Docker image generated by the installer. It is recommended that you accept the default value 1 hour.
 
+### <a name="enable-docker-gc"></a>enable_docker_gc
+This parameter specifies whether to run the [docker-gc](https://github.com/spotify/docker-gc#excluding-images-from-garbage-collection) script, a simple Docker container and image garbage collection script, once every hour to clean up stray Docker containers. You can configure the runtime behavior by using the `/etc/` config. For more information, see the [documentation](https://github.com/spotify/docker-gc#excluding-images-from-garbage-collection)
```
Comment (cmaloney) [inline 1.8/administration/installing/custom/configuration-parameters.md]: I don't believe this applies to DC/OS 1.8 (I think it only made 1.9)

**Comment by cmaloney** [comment]
One thing but other than that looks like we should :ship:

File: 1.8/administration/installing/custom/configuration-parameters.md (hunk 1 above)
Comment (joel-hamill) [inline 1.8/administration/installing/custom/configuration-parameters.md]: okay, removed

**Pull request merged**